packages/dcos-history/extra/history/server.py

```diff
@@ -1,93 +1,8 @@
-import logging
 import os
-import sys
+import history.server_util
```

Comment:
The motivation for splitting this was we did not want to invoke the thread for unit testing, if name main doesnt work with gunicorn, not having if name main doesnt work for unit tests. Separating is the cleanest way of doing this. We unit test for the state buffer directly without invoking it using app